automaxlair/da_controller.py

```diff
@@ -578,15 +585,16 @@ def check_stats(self) -> bool:
                 self.get_frame(), self.speed_stat_rect, threshold=False,
                 segmentation_mode='--psm 8'
             )
-            for expected_speed in self.expected_speed_stats.split(','):
+            for ii, expected_speed in enumerate(self.expected_speed_stats):
+                expected_speed = str(expected_speed)
                 nature_plus_expected = False
                 nature_minus_expected = False
-                if '+' in expected_speed:
-                    nature_plus_expected = True
-                    expected_speed = expected_speed.strip('+')
-                elif '-' in expected_speed:
+                # first index is minus expected
```

Comment:
actually, there can be more than one stat expected for minus. (for tornadus for example)

Reply:
I've addressed this in the latest version, it's now a dictionary storing the values we want.